src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -34,6 +37,7 @@ public class QualifcationConfiguration
    {
       public string SnapshotPath { get; set; }
       public string OutputFolder { get; set; }
+      public string ObservedDataFolderName { get; set; } = CoreConstants.DEFAULT_QUALIFICATION_OBSERVED_DATA_FOLDER_NAME;
```

Comment:
This were the changes that I had when we talked yesterday. going to update with our new file system